src/style/blocks/CardList.css

```diff
@@ -0,0 +1,88 @@
+@import url(./StarsBlock.css);
+@import url(./../variables.css);
+
+.CardList {
+  font-family: "Roboto", sans-serif;
+
+}
+
+.Card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  padding: 0 16px 16px;
+
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-content: center;
+  justify-content: center;
+}
+
+.Card-Img {
+  padding-top: 32px;
+  padding-left: 3px;
+  padding-right: 3px;
+  width: 160px;
+  height: 134px;
+}
+
+.Card-Desc {
+  padding-top: 40px;
+  font-weight: 500;
+  color: var(--Dark-Blue);
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.Card-Code {
+  padding-top: 4px;
+
+  color: var(--Color-Grey);
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.Card-Price {
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 16px;
+}
+
+.Price-Text {
+  color: var(--Color-Grey);
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.Price-Amount {
+  color: var(--Dark-Blue);
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.Card-Button {
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  box-sizing: border-box;
+  background-color: var(--Blue-Color);
+  border: 1px solid var(--Blue-Color);
+
+  padding-top: 12px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+  text-transform: uppercase;
+
+  text-decoration: none;
+  text-align: center;
+}
+
+.Card-Button:hover {
+  background-color: #fff;
+  color: var(--Blue-Color);
+}
```

Comment:
You should check BEM naming rules at first

Reply:
> You should check BEM naming rules at first

I should rewrite BlockName-ElemName to block-name__elem-name?